src/macaron/__main__.py

```diff
@@ -76,15 +77,20 @@ def analyze_slsa_levels_single(analyzer_single_args: argparse.Namespace) -> None
     sys.exit(status_code)
 
 
-def verify_prov(verify_args: argparse.Namespace) -> None:
+def verify_prov(verify_args: argparse.Namespace) -> NoReturn:
     """Verify a provenance against a user defined policy."""
     prov_file = verify_args.provenance
-    policy_file = verify_args.policy
+    policy_files = list(filter(lambda path: ".yaml" == os.path.splitext(path)[1], global_config.policy_paths))
```

Comment:
True. I also added a todo under the policy class; I think eventually it would be better to have the Policy class define what filetypes it can be constructed from (eg. with a method that is passed the filename)  so the configuration can just ask each policy and pick the first one that says yes. Currently though cue and YAML policies both use the same class, and souffle policies are another unrelated class since they need different interfaces, but the design could still be rationalized better.